### PR TITLE
Make pending commit status yellow again

### PR DESCRIPTION
**Pull request by silverwind**
With the introduction of Actions, the pending commit icon has changed from yellow to grey for Drone integrations which never set the "running" status, so it stays in "pending" until completion.

I find it better to have this icon colored like on 1.19. Now both the "pending" and "running" icons look the same, but I guess we could add an animation to the "running" state similar to GitHub has to it later.

Before:
Screenshot 2023-07-17 at 19 14 19

After:
Screenshot 2023-07-17 at 19 14 30

Also, it matches GH's icon:

image

### CODE REVIEW

File: templates/repo/commit_status.tmpl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{if eq .State "pending"}}
-	{{svg "octicon-dot-fill" 18 "commit-status icon text grey"}}
+	{{svg "octicon-dot-fill" 18 "commit-status icon text yellow"}}
 {{end}}
 {{if eq .State "running"}}
```
Comment (delvh) [inline templates/repo/commit_status.tmpl]:
```suggestion
```

File: templates/repo/commit_status.tmpl [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {{if eq .State "pending"}}
```
Comment (delvh) [inline templates/repo/commit_status.tmpl]:
```suggestion
{{if or (eq .State "pending") (eq .State "running")}}
```

**Review by delvh — APPROVED** [review]

**Comment by silverwind** [comment]
I will look into adding an animation for running state separately, this alone here is good to backport.

**Comment by silverwind** [comment]
Tests need update 

```
--- FAIL: TestRepoCommitsWithStatusPending (1.67s)
    repo_commits_test.go:70: 
        	Error Trace:	/home/runner/work/gitea/gitea/tests/integration/repo_commits_test.go:70
        	            				/home/runner/work/gitea/gitea/tests/integration/repo_commits_test.go:113
        	Error:      	Should be true
        	Test:       	TestRepoCommitsWithStatusPending

**Review by a1012112796 — APPROVED** [review]

**Review by puni9869 — APPROVED** [review]

**Comment by wolfogre** [comment]
Since `pending` and `running` have the same shape and color, why do we need `running`?

Please review #25839

**Comment by silverwind** [comment]
It could be animated, at least on some pages.

**Pull request merged**